code/game/machinery/teleporter.dm

```diff
@@ -353,6 +353,8 @@
 	for(var/obj/item/stock_parts/matter_bin/M in component_parts)
 		A += M.rating
 	accurate = A
+	if(accurate >= 3)
```

Comment:
Here is >= 3 wheres below is <= 3.

What happens when you pass a `3` through here.

Reply:
fixed